types/unicode-trie/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for unicode-trie 2.0
+// Project: https://github.com/foliojs/unicode-trie
+// Definitions by: esheppard <https://github.com/esheppard>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+/// <reference path="./builder.d.ts" />
+/// <reference path="./swap.d.ts" />
```

Comment:
You can safely remove `swap`, as this is interal one IMO.
The builder should be probably removed from here as well, instead, please add proper test:

```ts
import UnicodeTrieBuilder = require('unicode-trie/builder');
```
this will make sure it's published with DT types.

Reply:
Thanks Peter, changes pushed.